README.md

```diff
@@ -13,15 +13,16 @@ See [here](docs/supported_inventory_types.md) for the list of currently supporte
 To build SCALIBR, you'll need to have the following installed:
 
 * `go`: Follow https://go.dev/doc/install
-* `protoc`: Install the appropriate package, e.g. `apt install protobuf-compiler`
+* `protoc`: Install the appropriate [precompiled protoc binary](https://grpc.io/docs/protoc-installation/#install-pre-compiled-binaries-any-os).
+  * For Mac, you can also [install via HomeBrew](https://grpc.io/docs/protoc-installation/#install-using-a-package-manager).
 * `protoc-gen-go`: Run `go install google.golang.org/protobuf/cmd/protoc-gen-go`
 
 
 ## How to use
 
 ### As a standalone binary
 
-1. `make`
+1. `make` (use `sudo make` to cleanup prior built proto without prompting)
```

Comment:
Hm, I'm not sure if that's something we should recommend people do, sounds a bit hacky to avoid the prompts by running as sudo

Reply:
Agreed, removed